r3bbase/R3BFileSource.cxx

```diff
@@ -357,6 +375,45 @@ Bool_t R3BFileSource::Init()
         fInputFile.seekg(0, std::ios::beg);
     }
 
+    if (fInputFile.is_open())
+    {
+        LOG(INFO) << "R3BFileSource::Init() Reading RunId file";
+        std::string buffer;
+        char* p;
+        do
+        {
+            getline(fInputFile, buffer);
```

Comment:
The return value of `getline` should be checked: If it returns -1, then something went wrong, and `errno` indicates the error.